lib/neo4j/railtie.rb

```diff
@@ -4,7 +4,7 @@
 require 'action_dispatch/railtie'
 
 module Neo4j
-  class Railtie < ::Rails::Railtie
+  class Railtie < Rails::Railtie
```

Comment:
Interesting...  What made it so that not specifying the root `Rails` module fixed a problem?  I would expect this to break if we later, for whatever reason, make a `Neo4j::Rails` module